src/common/filament.cpp

```diff
@@ -22,29 +22,17 @@ const char *get_selected_filament_name() {
 #endif                // USE_<display>
 
 // clang-format off
-// to keep the texts aligned for easier checking of their alignment on the LCD
-static constexpr const char *pla_str =      "PLA      " SPACES "215/ 60";
-static constexpr const char *pet_g_str =    "PETG     " SPACES "230/ 85";
-static constexpr const char *pet_g_nbh_str ="PETG_NH  " SPACES "230/  0";
-static constexpr const char *asa_str =      "ASA      " SPACES "260/100";
-static constexpr const char *pc_str =       "PC       " SPACES "275/100";
-static constexpr const char *pvb_str =      "PVB      " SPACES "215/ 75";
-static constexpr const char *abs_str =      "ABS      " SPACES "255/100";
-static constexpr const char *hips_str =     "HIPS     " SPACES "220/100";
-static constexpr const char *pp_str =       "PP       " SPACES "240/100";
-static constexpr const char *flex_str =     "FLEX     " SPACES "240/ 50";
-
 const Filaments::Array filaments = {
-    { "---", BtnResponse::GetText(Response::Cooldown),   0,    0,   0, Response::Cooldown }, // Cooldown sets long text instead short, not a bug
-    { BtnResponse::GetText(Response::PLA), pla_str,     215, 170,  60, Response::PLA },
-    { BtnResponse::GetText(Response::PETG), pet_g_str,  230, 170,  85, Response::PETG },
-    { BtnResponse::GetText(Response::ASA), asa_str,     260, 170, 100, Response::ASA },
-    { BtnResponse::GetText(Response::PC), pc_str,       275, 170, 100, Response::PC },
-    { BtnResponse::GetText(Response::PVB), pvb_str,     215, 170,  75, Response::PVB },
-    { BtnResponse::GetText(Response::ABS), abs_str,     255, 170, 100, Response::ABS },
-    { BtnResponse::GetText(Response::HIPS), hips_str,   220, 170, 100, Response::HIPS },
-    { BtnResponse::GetText(Response::PP), pp_str,       240, 170, 100, Response::PP },
-    { BtnResponse::GetText(Response::FLEX), flex_str,   240, 170,  50, Response::FLEX },
+    { "---",                                  0,   0,   0, Response::Cooldown }, // Cooldown sets long text instead short, not a bug
```

Comment:
`long_name` was removed, the comment is no longer relevant.

Reply:
I'll fix that, thx